tests/v3/test_store/test_remote.py

```diff
@@ -2,6 +2,7 @@
 
 import fsspec
 import pytest
+from upath import UPath
```

Comment:
add to test requirements, or use importorskip in affected tests

Reply:
added to test requirements